Podfile.lock

```diff
@@ -785,7 +785,7 @@ SPEC CHECKSUMS:
   DoubleConversion: e22e0762848812a87afd67ffda3998d9ef29170c
   Down: 71bf4af3c04fa093e65dffa25c4b64fa61287373
   FBLazyVector: 2bf7b5e351f8e33867210ff6eb9c5c178a035522
-  FBReactNativeSpec: 5a1f15997f42c2f266b63a6b9c3e92a9eca049f5
+  FBReactNativeSpec: b4076f780f69b07abd859b279490f401d127c0ec
```

Comment:
In the CircleCI days, we used to [guard](https://github.com/wordpress-mobile/circleci-orbs/blob/25eee7663ad3d9fc908c7602ae3262379ea33c61/src/ios/orb.yml#L519-L552) against this kind of issues in the lockfile.

Unless I'm missing something, I don't see a similar logic in Buildkite. If that's the case, we should port it over because it would have most likely caught this issue.